drivers/xen/consfront/Config.uk

```diff
@@ -1,8 +1,8 @@
-menuconfig LIBXEN_CONSOLE
-	bool "Hypervisor console"
+config LIBCONSFRONT
+	bool "consfront"
```

Comment:
I would make it LIBXENCONSOLE. That's because there is no such entity as xenconsback. Blk, net, 9p are calling fronts because there are backends for them. Such backends may be called as process which is running in some domain. Typically DomD and you can configure domain with running backend. For 9p it's qemu process.
Same for sndbe and displbe.

Reply:
Funnily, in Xen's Mini-OS, they named the driver [`consfront`](https://xenbits.xen.org/gitweb/?p=mini-os.git;a=tree), they also have `fbfront` for a FrameBuffer/Keyboard driver - they got completely convised with the `*front` naming, independent to if there is a backend. However, their [Windows PV drivers](https://xenbits.xen.org/gitweb/?a=project_list;pf=pvdrivers/win) are named with `xencons`, `xennet`, `xenblk`... So, I must say I like your reasoning and would adopt it to `LIBXENCONS`.